src/pip/_internal/vcs/subversion.py

```diff
@@ -33,6 +37,28 @@ def should_add_vcs_url_prefix(cls, remote_url):
     def get_base_rev_args(rev):
         return ['-r', rev]
 
+    def get_vcs_version(self):
+        # type: () -> Tuple[int, ...]
+        """Return the version of the currently installed Subversion client.
+
+        :return Tuple containing the parts of the version information.
+        :raises BadCommand: If ``svn`` is not installed.
+            ValueError: If the version returned from ``svn`` could not be
+            parsed.
+        """
+        version_pfx = 'svn, version '
+        version = self.run_command(['--version'], show_stdout=False)
```

Comment:
Can you include one of your sample 2-line outputs as a code comment here? That would help people to know what the full output being parsed looks like.